### DISCUSSION TITLE
match typo

### DISCUSSION THREAD

**Discussion by DanielAjoy** [h3rald/min#170] [Bugs: Fixed]
https://min-lang.org/learn-definitions/

talks about **match** here:

```
 dup dup 
 "\.zip$" match 
 swap fsize 1000000 > and 
 swap mtime now 3600 - >
```

should be **match?**

**Comment by h3rald** [comment]
Yeah, it should. It used to be `match`, but I renamed it for consistency with other predicates later on. I must have missed this example. Fixed!
